docs/serving/observability/metrics/serving-metrics.md

```diff
@@ -5,105 +5,164 @@ components:
 function: reference
 ---
 
-# Knative Serving metrics
+# Knative Serving Metrics
 
 Administrators can monitor Serving control plane based on the metrics exposed by each Serving component.
-Metrics are listed next.
+
+!!! note
+
+    These metrics may change as we flush out our migration from OpenCensus to OpenTelemetry
+
+## Queue Proxy
+
+The queue proxy is the per-pod sidecar that enforces container concurrency and provides metrics to the autoscaler. The following metrics provide you insights into queued
+requests and user-container behavior.
+
+###  `kn.queueproxy.depth`
+
+**Instrument Type:** Int64Gauge
+
+**Unit (UCUM):** {item}
+
+**Description:** Number of current items in the queue proxy queue
+
+### `kn.queueproxy.app.duration`
+
+**Instrument Type:** Float64Histogram
+
+**Unit (UCUM):** s
+
+**Description:** The duration of the task execution
 
 ## Activator
 
 The following metrics can help you to understand how an application responds when traffic passes through the activator. For example, when scaling from zero, high request latency might mean that requests are taking too much time to be fulfilled.
 
-| Metric Name | Description | Type | Tags | Unit | Status |
-|:-|:-|:-|:-|:-|:-|
-| ```request_concurrency``` | Concurrent requests that are routed to Activator<br>These are requests reported by the concurrency reporter which may not be done yet.<br> This is the average concurrency over a reporting period | Gauge | ```configuration_name```<br>```container_name```<br>```namespace_name```<br>```pod_name```<br>```revision_name```<br>```service_name``` | Dimensionless | Stable |
-| ```request_count``` | The number of requests that are routed to Activator.<br>These are requests that have been fulfilled from the activator handler. | Counter | ```configuration_name```<br>```container_name```<br>```namespace_name```<br>```pod_name```<br>```response_code```<br>```response_code_class```<br>```revision_name```<br>```service_name``` | Dimensionless | Stable |
-| ```request_latencies``` | The response time in millisecond for the fulfilled routed requests | Histogram | ```configuration_name```<br>```container_name```<br>```namespace_name```<br>```pod_name```<br>```response_code```<br>```response_code_class```<br>```revision_name```<br>```service_name``` | Milliseconds | Stable |
+
+### `kn.revision.request.concurrency`
+
+**Instrument Type:** Float64Gauge
+
+**Unit (UCUM):** {request}
+
+**Description:** Concurrent requests that are routed to the Activator
+
+The following attributes are included with the metrics below
+
+Name | Type | Description
+-|-|-
+`k8s.namespace.name` | string | Namespace of the Revision
+`kn.service.name` | string | Knative Service name associated with this Revision
+`kn.configuration.name` | string | Knative Configuration name associated with this Revision
+`kn.revision.name` | string | The name of the Revision
```

Comment:
I wasn't certain about this format vs the previous tabular format, but reviewing it on the site preview, it seems pretty reasonable.  I sort of want a database of these, but I don't think I want to invest in that enough to make it happen.

Reply:
I'm finding the navigation bar on the left and the sidebar on the right make it harder to read wide tables due to word wrapping.